GP/gpiConnect.c

```diff
@@ -336,8 +336,8 @@ gpiSendLogin(GPConnection * connection,
 	GPIConnection * iconnection = (GPIConnection*)*connection;
 	GPIProfile * profile;
 	char * passphrase;
-	char userBuffer[GP_NICK_LEN + GP_EMAIL_LEN];
 	char partnerBuffer[11];
```

Comment:
I would do a macro GP_PARTNETBUFFER_LEN 11 and change this line to GP_NICK_LEN + GP_EMAIL_LEN + GP_PARTNERBUFFER_LENGTH + 1

Reply:
https://github.com/GameProgressive/UniSpySDK/pull/27/commits/45fbbc5847c681a8cacaa983dafacdcc9627c4c8